ocp_resources/image_digest_mirror_set.py

```diff
@@ -0,0 +1,28 @@
+from ocp_resources.resource import Resource
+
+
+class ImageDigestMirrorSet(Resource):
+    """
+    Create/Manage ImageDigestMirrorSet configuration object. API reference:
+    https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html
+    Args:
```

Comment:
In class dockstring: `API reference: https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html`

in __init__: 
```
Create/Manage ImageDigestMirrorSet configuration object

Args: .....
```

:)

Reply:
Sorry! Misunderstood. Will change.